### ISSUE TITLE
Add support for RTL languages

### ISSUE THREAD

**Issue by NaifAlrashed** [kean/Pulse#114]
when opening the ui in right to left languages, we have problems reading the interface. I included a pic for the most painful page, the viewer
![IMG_7DC51DE45E89-1](https://user-images.githubusercontent.com/15128710/190903159-bc97236a-8fe8-4ca6-9702-97b3db990923.jpeg)

**Comment by kean** [comment]
Thanks for the report. PRs are welcome.

**Comment by kean** [comment]
I've added a workaround for the JSON viewer in 2.2.0 where I hardcoded the language direction, so at least it's readable.

![Screenshot 2022-12-30 at 7 57 01 PM](https://user-images.githubusercontent.com/1567433/210120275-3c5b7733-2d09-4143-bcb7-cb417cd55680.png)

Unfortunately, I don't have experience supporting RTL languages. The app is also not localized and supports only English at the moment. I would appreciate some help adding proper support for RTL interfaces, and I'm open to adding localization support.
